src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -1946,33 +1831,106 @@ public with sharing class ADDR_Addresses_TEST {
 
     /*********************************************************************************************************
     @description
-        insert enough addresses to require multiple trigger calls, to ensure our static counters are handled
-        appropriately.
+        Insert more than 200 addresses to require multiple trigger calls, and to ensure our static counters 
+        are handled appropriately.
     verify:
-        address propagated to all contacts and accounts
+        Addresses are propagated to all contacts and accounts
     **********************************************************************************************************/
-    static testMethod void insertManyAddresses() {
-        if (strTestOnly != '*' && strTestOnly != 'insertManyAddresses') return;
+    @isTest
+    private static void addressShouldBePropagatedToAccountsAndContactsOnMultipleTriggerCalls() {
```

Comment:
@force2b , the [TDTM_DMLgt200_TEST.test_addresses()](https://github.com/SalesforceFoundation/Cumulus/blob/master/src/classes/TDTM_DMLgt200_TEST.cls#L325) could not be reused instead of this test since this test is creating Accounts, Contacts and then Addresses ensuring when the Address is assigned to an Account, it is propagated to the Account and its Contacts. The `TDTM_DMLgt200_TEST.test_addresses()` is testing Account and Address creation when Contacts are inserted.
Let me know if the above is not correct.

Reply:
Difference is styling in the class, and in this method, usage of `UTIL_UnitTestData_TEST.disable*Triggers()` and `UTIL_Debug.disableDebugLogging()`. No other functional changes are applied, however, since the CPU Timeout error is intermittent, we do not know if actually this work when builds are being created.
Creating a separate class using `testSetup` is good idea. Shall I go ahead with its implementation?

Reply:
Great point about disabling the triggers. Might add a slight improvement. I'm fine with this as is for now, but I like knowing that we have an alternative path to try if still get errors. 

What I've been doing to test this is use the `/rebuild` option to force a rebuild on this branch. I'm hoping that that if we do that a whole bunch of times it won't fail at all. It just takes an hour between each retry. If it does eventually fail with a time out, then we can try a different class. I just did a second time (for a total of 3 builds). When it's done, just try it again (and again, etc.). to see what happens.

Reply:
@force2b , the rebuilds have been run (5 builds in [43157](https://mrbelvedereci.herokuapp.com/builds/43157/rebuilds) (ignore the Queued) and 5 builds in [43531](https://mrbelvedereci.herokuapp.com/builds/43531/rebuilds)). Tests passed. 
Do you think more rebuilds should be run? If not, is there anything else outstanding for the PR that I needs to be fixed?